third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -696,6 +699,45 @@ struct AtomicRMWOpConversion
            (elementType.isF16() || elementType.isBF16() || elementType.isF32());
   }
 
+  bool isPromotableToNVPTXLD(triton::AtomicRMWOp op) const {
+    if (disableLDAcquireLowering)
+      return false;
+
+    Type valueTy =
+        getTypeConverter()->convertType(getElementTypeOrSelf(op.getType()));
+
+    if (!valueTy.isIntOrFloat() || valueTy.getIntOrFloatBitWidth() != 32)
+      return false;
+    if (op.getSem() != triton::MemSemantic::ACQUIRE)
```

Comment:
Why not support relaxed as well?

Reply:
Only acquire was needed at the moment, for the purpose of improving atomic_add(0) speed but I am considering adding relaxed as well.

Reply:
Added relaxed back in.